public/app/features/explore/PrometheusListView/RawListItem.tsx

```diff
@@ -0,0 +1,162 @@
+  for (const key in allLabels) {
+    if (key in allLabels && allLabels[key] && !key.includes('Value')) {
+      attributeValues.push({
+        key: key,
+        value: allLabels[key],
+      });
+    } else if (key in allLabels && allLabels[key] && key.includes('Value')) {
+      values.push({
+        key: key,
+        value: allLabels[key],
+      });
+    }
+  }
+  return {
+    values: values,
+    attributeValues: attributeValues,
+  };
+}
+
+const RawListItem = ({ listItemData, listKey, totalNumberOfValues, valueLabels, isExpandedView }: RawListProps) => {
+  const { __name__, ...allLabels } = listItemData;
+  const [_, copyToClipboard] = useCopyToClipboard();
+  const displayLength = valueLabels?.length ?? totalNumberOfValues;
+  const styles = useStyles((theme) => getStyles(theme, displayLength, isExpandedView));
+  const { values, attributeValues } = getQueryValues(allLabels);
+
+  /**
+   * Transform the symbols in the dataFrame to uniform strings
+   */
+  const transformCopyValue = (value: string): string => {
+    if (value === '∞') {
+      return '+Inf';
+    }
+    return value;
+  };
+
+  // Convert the object back into a string
+  const stringRep = `${__name__}{${attributeValues.map((value) => {
+    // For histograms the string representation currently in this object is not directly queryable in all situations, leading to broken copied queries. Omitting the attribute from the copied result gives a query which returns all le values, which I assume to be a more common use case.
+    return value.key !== 'le' ? `${value.key}="${transformCopyValue(value.value)}"` : '';
+  })}}`;
+
+  const hideFieldsWithoutValues = Boolean(valueLabels && valueLabels?.length);
+
+  return (
+    <>
+      {valueLabels !== undefined && isExpandedView && (
+        <ItemLabels valueLabels={valueLabels} expanded={isExpandedView} />
+      )}
+      <div key={listKey} className={styles.rowWrapper}>
+        <span className={styles.copyToClipboardWrapper}>
+          <IconButton tooltip="Copy to clipboard" onClick={() => copyToClipboard(stringRep)} name="copy" />
+        </span>
+        <span className={styles.rowLabelWrapWrap}>
+          <div className={styles.rowLabelWrap}>
+            <span>{__name__}</span>
+            <span>{`{`}</span>
+            <span>
+              {attributeValues.map((value, index) => (
+                <RawListItemAttributes
+                  isExpandedView={isExpandedView}
+                  value={value}
+                  key={index}
+                  index={index}
+                  length={attributeValues.length}
+                />
+              ))}
+            </span>
+            <span>{`}`}</span>
+          </div>
+        </span>
+
+        {/* Output the values */}
+        <ItemValues
+          hideFieldsWithoutValues={hideFieldsWithoutValues}
+          totalNumberOfValues={displayLength}
+          values={values}
+        />
+      </div>
+    </>
+  );
+};
+export default RawListItem;
```

Comment:
do you mind adding aria tags as well? For better a11y.

Reply:
I added labels for the "cells" of this "table", which is how this UI should be consumed by a screen reader AFAIK, I also removed the expand/contract button from the screen reader API as its functionality is completely useless. Am I missing anything?

Reply:
I meant not only this component but the whole changeset. Let me go through again and see if I will find anything